src/Resources/contao/classes/DataContainer.php

```diff
@@ -447,6 +466,45 @@ protected function row($strPalette=null)
   </script>';
 		}
 
+		// Page picker
+		if ($arrData['eval']['dcaPicker'])
```

Comment:
I have changed this in the latest PR, however we now end up with `'dcaPicker'=>array()` most of the time, which feels a little unfamiliar.